main.go

```diff
@@ -162,6 +164,62 @@ func dumpJSONFile(file string, pretty bool, schema bool) error {
 	return nil
 }
 
+func getFilesContentHash(files []string, tmp string) (string, bool, error) {
+	sha1 := sha1.New()
```

Comment:
Do you want to sort the files here?

Reply:
But you'd need to sort a copy of this files list because order is important in the query.

Reply:
Uhm we would also to sort the table order in the query as well. Because the order in the database is already set. Like if our initial call is 
./dsq --cache  testdata/join/ages.json testdata/join/users.csv  "select u.name, a.age from {1} u join {0} a on u.id = a.id"
Our second call of:
./dsq --cache testdata/join/users.csv  testdata/join/ages.json   "select u.name, a.age from {0} u join {1} a on u.id = a.id"
wouldn't work.

Reply:
Would sorting that too has unintended side effects?

Reply:
I am testing it with the copy, the order in the query wouldn't be correct because the order in the sqlite file is according to the first query

Reply:
Oh yeah that's a good point. OK! No need for sorting